api/config/config_prod.json

```diff
@@ -54,6 +54,6 @@
     "enableComplianceTraining": false,
     "enableEraCommons": false,
     "enableDataUseAgreement": false,
-    "enableIdVerification": false
+    "enableIdVerification": true
```

Comment:
Why is `enableDataUseAgreement` also false? Is that correct?

Reply:
I don't believe we have anything in dataUseAgreement yet. It should be getting set though, so I don't believe we need it false?

Reply:
As far as I know we don't have that module enabled yet?

Reply:
Codewise I am seeing it not enabled. I think we used to autopopulate as part of the unregistered page? (@calbach correct me if I am wrong) but I don't think we lose anything by turning it off. From what I can see in the DB, the two things autopopulated are:
`terms_of_service_completion_time`
`demographic_survey_completion_time`

Reply:
I think we lost the `terms_of_service_completion_time` at some point though, I don't see it getting read anywhere in UserService

Reply:
Yeah, this is fine, I wasn't clear on whether this was being implemented with the same field or a new one. Looks like it's a totally new field. Also the code is commented out so this wouldn't make much of a different. `false` should be the correct value for now then.